app/src/main/java/org/hackillinois/android/database/entity/Event.kt

```diff
@@ -35,6 +35,22 @@ data class Event(
         }
     }
 
+    fun getEndTimeOfDay(): String {
```

Comment:
Can you make a function called getTimeOfDay() which can be used inside of getStartTimeOfDay() and getEndTimeOfDay()? Never copy and paste codeeeee

Reply:
Ooops sorry